### PR TITLE
Fix build to enable Angular v9 support

### PR DESCRIPTION
**Pull request by petebacondarwin**
The current library on npm contains both `@angular/common`
and `rxjs`  in the distributable files. Apart from meaning that
these files are vastly larger than they need to be it also causes
the Angular `ngcc` compilation to fail, which prevents this
library from being used in an app compiled with Angular ivy.

This commit adds `rxjs` and `@angular/common` as externals
to the rollup configuration which solves this problem.

### CODE REVIEW

File: src/tsconfig.es5.json [hunk 1]
```diff
@@ -7,8 +7,7 @@
     "types": []
   },
   "files": [
-    "./index.ts",
-    "./typings.d.ts"
```
Comment (petebacondarwin) [inline src/tsconfig.es5.json]: This typings file does not exist.

File: src/tsconfig.spec.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
-  "extends": "../../tsconfig.json",
```
Comment (petebacondarwin) [inline src/tsconfig.spec.json]: It looks like these files used to be under the `lib` folder.

**Comment by jaibatrik** [comment]
Thanks for the PR. Does this mean it will now be compatible with Ivy as well?

**Comment by petebacondarwin** [comment]
From my local testing, it appears that your library is now compatible with Ivy 🎉 
Please can you publish a new version of the library.

**Comment by jaibatrik** [comment]
Sure, I'll do it today / tomorrow. Thanks again!

**Comment by jaibatrik** [comment]
I have released npm version 2.2.0 with the changes. Thank you!

**Comment by petebacondarwin** [comment]
🎉  I can confirm it is now working on our ngcc-validation repo.

**Pull request merged**